cfg/rippled-example.cfg

```diff
@@ -1212,23 +1314,24 @@ medium
 
 # This is primary persistent datastore for rippled.  This includes transaction
 # metadata, account states, and ledger headers.  Helpful information can be
-# found here: https://ripple.com/wiki/NodeBackEnd
-# delete old ledgers while maintaining at least 2000. Do not require an
-# external administrative command to initiate deletion.
+# found at https://xrpl.org/capacity-planning.html#node-db-type
+# type=NuDB is recommended for non-validators with fast SSDs. Validators or
```

Comment:
I would use a `none` database backend on a validator. They don't need to concern themselves with historic data at all.

Reply:
The linked capacity planning page recommends RocksDB, but I can definitely see your point about using `none` (or `memory`). My biggest concern with using those options on the individual node level, is that it means it will take much longer to start and restart, because the node will have to download the entire ledger every time.  On the global level, I'm thinking about a failsafe / worst case situation. For example, if every validator shuts down simultaneously :scream:, they will need to have some data on disk so they can bootstrap restarting the network.

Reply:
Anecdotally, my full history node takes about half an hour to start with `--load` vs. a few minutes at most to get synced with `--net`.

Bootstrapping can be done safely with `--ledgerfile` (https://xrpl.org/commandline-usage.html#initial-ledger-options) and that's probably better anyways a better way forward than trusting that data was stored to disk correctly in such a catastrophic scenario. The file itself could be hashed and communicated out-of-band in a lot of different ways compared to a node database that's not even a full shard in most cases.

Reply:
@MarkusTeufelberger: re: slow start times with `--load`: are you using SSDs?

Reply:
Of course. Over a dozen TB of 'em.

Reply:
I think that `--net` should become the default now, unless a different command line option (for which I don't have a good name) is specified.

Reply:
@nbougalis I think making `--net` the default is out of scope for this PR. Do you agree?